prdoc/pr_8179.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Do not make pallet-identity benchmarks signature-dependent
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      - Includes a `BenchmarkHelper` configuration item in `pallet-identity` to handle signing operations.
+      - Abstracts away the explicit link with Sr25519 schema in the benchmarks, allowing chains with a different one to be able to run them and calculate the weights.
+      - Adds a default implementation for the empty tuple that leaves the code equivalent.
```

Comment:
```suggestion
      - Adds a default implementation for the empty tuple that leaves the code equivalent.
      
      Adding the following to your implementation of the `frame_identity::Config` should be enough:
        ```diff
        #[cfg(feature = "runtime-benchmarks")]
        type BenchmarkHelper = ();
        ```
```

Reply:
Done in [63c1e13](https://github.com/paritytech/polkadot-sdk/pull/8179/commits/63c1e13147737802a9cb155f2d77d48e52577f32).